AltStore/Operations/ResignAppOperation.swift

```diff
@@ -113,11 +113,21 @@ private extension ResignAppOperation
             
             infoDictionary[kCFBundleIdentifierKey as String] = profile.bundleIdentifier
             infoDictionary[Bundle.Info.altBundleID] = identifier
-            
+
             for (key, value) in additionalInfoDictionaryValues
             {
                 infoDictionary[key] = value
             }
+
+            if var extensionInfo = infoDictionary["NSExtension"] as? [String: Any],
+                let extensionType = extensionInfo["NSExtensionPointIdentifier"] as? String,
+                let appGroup = extensionInfo["NSExtensionFileProviderDocumentGroup"] as? String,
+                let teamId = profile.entitlements[.teamIdentifier] as? String,
+                extensionType == "com.apple.fileprovider-nonui"
```

Comment:
Do UI FileProviders (`com.apple.fileprovider-ui`) also need this change? Or just non-UI FileProviders?

Reply:
No idea, but I decided to stop checking the extension point, because NSExtensionFileProviderDocumentGroup is going to be an app group regardless of what the extension point might be.